### PR TITLE
Minor fixes and improvements to the documentation pages

### PR DESCRIPTION
**Pull request by barfowl**
This set of changes includes fixes and improvements in three areas:

- pages for the examples were updated to reflect recent command line argument conventions
- broken links were updated to reflect renamed sections or removed if no longer relevant
- source for obsolete documentation pages and images were removed

### CODE REVIEW

**Comment by c64kernal** [comment]
This is really great, thanks @barfowl -- I left some very minor notes, let me know what you think and I can merge after.  Thanks again!

File: documentation/code_examples.rst [hunk 1]
```diff
@@ -54,10 +54,35 @@ of the software.
...
+Common Command Line Options
+===========================
+
+While the following command line options are common, not all examples support
+all -- in some cases, certain options are not applicable.
+
+::
+
+      -f         : launches the application in full-screen mode (if supported by GLFW
+                   on the OS)
```
[the remark this answers is not in the document]
Comment (barfowl) [reply documentation/code_examples.rst]: No objection -- I simply cut/pasted this text from its usage in the individual examples.

File: documentation/code_examples.rst [hunk 2]
```diff
@@ -54,10 +54,35 @@ of the software.
...
+      -u         : apply uniform subdivision to all meshes
+      -a         : apply adaptive subdivision to create patches for the limit surface
+      -l depth   : level of uniform or adaptive refinement to apply
+      -c count   : number of repetitions of the animation loop when supported (default
+                   of 0 is infinite)
+      objfile(s) : a set of one or more meshes in Obj format (requiring a .obj suffix)
```
[the remark this answers is not in the document]
Comment (barfowl) [reply documentation/code_examples.rst]: Agreed regarding use of "extension".
The argument ordering placed all arguments that apply specifically to given Obj arguments after the description of the Obj argument -- so the latter half are essentially a group of Obj related options.  Those that precede the Obj group are more widely used.  Moving "objfiles" to the end would serve the same purpose.

File: documentation/code_examples.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (barfowl) [reply documentation/code_examples.rst]: On second thought, how about we just go with "(if supported)"?

[the remark this answers is not in the document]
Comment (c64kernal) [reply documentation/code_examples.rst]: "(if supported)" is a great idea!

File: documentation/code_examples.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (c64kernal) [reply documentation/code_examples.rst]: Got it, that makes sense, thanks @barfowl!

**Comment by c64kernal** [comment]
Looks great, thanks so much @barfowl!

**Pull request merged**